<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>739</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QPlainTextEdit" name="ptxtEditRecv">
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>20</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>控制面板</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>波特率</string>
        </property>
        <property name="buddy">
         <cstring>cBoxBaudRate</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QComboBox" name="cBoxBaudRate"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>停止位</string>
        </property>
        <property name="buddy">
         <cstring>cBoxStopBit</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QComboBox" name="cBoxPortSelect"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>串口选择</string>
        </property>
        <property name="buddy">
         <cstring>cBoxPortSelect</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="QComboBox" name="cBoxStopBit"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>数据位</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>校验位</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QPushButton" name="btnOpenPort">
        <property name="text">
         <string>打开串口</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="2">
       <widget class="QComboBox" name="cBoxDataBit"/>
      </item>
      <item row="4" column="1" colspan="2">
       <widget class="QComboBox" name="cBoxParityBit"/>
      </item>
      <item row="6" column="2">
       <widget class="QPushButton" name="btnClearData">
        <property name="text">
         <string>清除接受</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0" colspan="3">
       <widget class="QCheckBox" name="chkBoxHexDisplay">
        <property name="text">
         <string>16进制显示</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="2">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>串口操作</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0" colspan="2">
       <widget class="QPushButton" name="btnSaveData">
        <property name="text">
         <string>保存数据</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPlainTextEdit" name="ptxtEditSend"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="btnSend">
         <property name="text">
          <string>发送</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btnClearSend">
         <property name="text">
          <string>清除发送</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="2">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QCheckBox" name="chkBoxScheduleSend">
         <property name="text">
          <string>定时发送</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>周期：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit">
         <property name="maximumSize">
          <size>
           <width>45</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>1000</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>ms</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_2"/>
       </item>
       <item>
        <widget class="QPushButton" name="btnOpenFile">
         <property name="text">
          <string>打开文件</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnSendFile">
         <property name="text">
          <string>发送文件</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnStopSend">
         <property name="text">
          <string>停止发送</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QCheckBox" name="chkBoxNewLine">
       <property name="text">
        <string>发送新行</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
